crates/ark/src/lsp/completions/sources/unique/custom.rs

```diff
@@ -9,8 +9,8 @@ use anyhow::Result;
 use harp::exec::RFunction;
 use harp::exec::RFunctionExt;
 use harp::object::RObject;
-use harp::utils::sym_quote_invalid;
 use harp::utils::r_typeof;
+use harp::utils::sym_quote_invalid;
```

Comment:
Why did it move? 🤔

Reply:
No idea! The formatter wants it there though.